test/e2e_test.go

```diff
@@ -136,6 +167,22 @@ var _ = Describe("MetricsServer", func() {
 		usage := ms.Containers[0].Usage
 		Expect(usage.Memory().Value()/1024/1024).To(BeNumerically("~", 50, 5), "Unexpected value of memory")
 	})
+	It("returns accurate memory metric", func() {
```

Comment:
Please merge this case so we have one test for init container. THere is no need to test accuracy of metric value as we already have a dedicate test for it. We just need to verify if we return metrics for when init container is present